src/bin/miri.rs

```diff
@@ -44,22 +44,25 @@ impl<'a> CompilerCalls<'a> for MiriCompilerCalls {
             fn extract_str(lit: &syntax::ast::Lit) -> syntax::parse::token::InternedString {
                 match lit.node {
                     syntax::ast::LitKind::Str(ref s, _) => s.clone(),
-                    _ => panic!("attribute values need to be strings"),
+                    _ => bug!("attribute values need to be strings"),
```

Comment:
@oli-obk It turns out this changed in a recent nightly. Beta still bails on non-string literals in attributes, but nightly accepts them if you use `#![feature(attr_literals)]`.